.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: groovy
 
 jdk:
-  - oraclejdk7
+  - oraclejdk8
```

Comment:
yep it's great time to leave java7 compatibility, pls update Compatible with jdk1.7+. README section